GitExtUtils/GitUI/PropertyGridExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Reflection;
+
+namespace GitExtUtils.GitUI;
+
+public static class PropertyGridExtensions
+{
+    public static void SetLabelColumnWidth(this PropertyGrid grid, int width)
+    {
+        FieldInfo fi = grid.GetType().GetField("_gridView", BindingFlags.Instance | BindingFlags.NonPublic);
+        Control view = fi?.GetValue(grid) as Control;
+        MethodInfo mi = view?.GetType().GetMethod("MoveSplitterTo", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
[perf] This is quite expensive - not only we'd be iterating over the type metadata, we'd be doing it for every invocation.

Also, we should now be favouring [`UnsafeAccessorAttribute`](https://learn.microsoft.com/dotnet/api/system.runtime.compilerservices.unsafeaccessorattribute) over the old-fashioned direct reflection calls.